.azure/scripts/setup-kubernetes.sh

```diff
@@ -88,14 +88,20 @@ if [ "$TEST_CLUSTER" = "minikube" ]; then
 
       set -ex
     fi
-    
+
     if [ "$ARCH" = "s390x" ] || [ "$ARCH" = "ppc64le" ]; then
         git clone -b v1.9.11 --depth 1 https://github.com/kubernetes/kubernetes.git
         sed -i 's/:1.11//' kubernetes/cluster/addons/registry/images/Dockerfile
         docker build --pull -t gcr.io/google_containers/kube-registry-proxy:0.4-${ARCH} kubernetes/cluster/addons/registry/images/
         minikube cache add ${ARCH}/registry:2.8.0-beta.1 gcr.io/google_containers/kube-registry-proxy:0.4-${ARCH}
         minikube addons enable registry --images="Registry=${ARCH}/registry:2.8.0-beta.1,KubeRegistryProxy=google_containers/kube-registry-proxy:0.4-${ARCH}"
-	rm -rf kubernetes
+        rm -rf kubernetes
+    elif [[ "$ARCH" = "arm64" ]]; then
+        git clone -b v1.9.11 --depth 1 https://github.com/kubernetes/kubernetes.git
+        sed -i 's/:1.11//' kubernetes/cluster/addons/registry/images/Dockerfile
+        minikube image build -t google_containers/kube-registry-proxy:0.5-SNAPSHOT kubernetes/cluster/addons/registry/images/
+        minikube addons enable registry --images="Registry=arm64v8/registry:2.8.2,KubeRegistryProxy=google_containers/kube-registry-proxy:0.5-SNAPSHOT"
+        rm -rf kubernetes
```

Comment:
Wouldn't this be quite an expensive operation? Would it make sense to have this somewhere out-of-band?

Reply:
Well IBM folks uses same approach as well in same script. So we followed same approach and it is really fast build of this image registry image.

Reply:
My concern would be more about the length of the cloning and GitHub availability. But we can give it a try and see if it causes issues.